ChatQnA/Dockerfile.wrapper

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Why did you need this one? We default use no wrapper mode for current examples.

Reply:
For remote TGI or vLLM inference endpoints if there is oAuth authentication flow we need to get the token and pass it in the headers. Currently this functionality is added in wrappers.

Reply:
Did you use docker image build by this dockerfile in this PR?

Reply:
If yes, you should add into the docker_image_build/build.yaml, and give it a new image name.

Reply:
yes added into the build.yaml file